cms/static/js/factories/manage_users.js

```diff
@@ -9,7 +9,7 @@ function(_, gettext, ManageUsersAndRoles) {
             var local_messages = _.extend({}, messages);
             local_messages.alreadyMember.title = gettext('Already a course team member');
             local_messages.deleteUser.messageTpl = gettext(
-                'Are you sure you want to delete {email} from the course team for “{container}”?'
+                'Are you sure you want to delete <%= email %> from the course team for “<%= container %>”?'
```

Comment:
Just wanted to remind you, cause it's the kind of thing that's easy to forget. I didn't actually trace this value all the way to the DOM, so I'll take your word for it.